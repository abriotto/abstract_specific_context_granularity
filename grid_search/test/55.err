  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  8%|###############1                                                                                                                                                                            | 8/99 [00:00<00:01, 63.40it/s] 15%|############################3                                                                                                                                                              | 15/99 [00:00<00:01, 54.86it/s] 22%|#########################################5                                                                                                                                                 | 22/99 [00:00<00:01, 58.42it/s] 29%|######################################################7                                                                                                                                    | 29/99 [00:00<00:01, 57.09it/s] 35%|##################################################################1                                                                                                                        | 35/99 [00:00<00:01, 57.83it/s] 41%|#############################################################################4                                                                                                             | 41/99 [00:00<00:01, 50.91it/s] 48%|##########################################################################################6                                                                                                | 48/99 [00:00<00:00, 52.47it/s] 61%|#################################################################################################################3                                                                         | 60/99 [00:01<00:00, 66.24it/s] 69%|################################################################################################################################4                                                          | 68/99 [00:01<00:00, 67.28it/s] 76%|#############################################################################################################################################6                                             | 75/99 [00:01<00:00, 66.33it/s] 84%|############################################################################################################################################################7                              | 83/99 [00:01<00:00, 65.34it/s] 91%|##########################################################################################################################################################################                 | 90/99 [00:01<00:00, 62.47it/s] 99%|#########################################################################################################################################################################################1 | 98/99 [00:01<00:00, 63.79it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 60.96it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 32%|############################################################1                                                                                                                               | 8/25 [00:00<00:00, 69.30it/s] 60%|################################################################################################################2                                                                          | 15/25 [00:00<00:00, 61.01it/s] 96%|###################################################################################################################################################################################5       | 24/25 [00:00<00:00, 65.95it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 65.02it/s]
