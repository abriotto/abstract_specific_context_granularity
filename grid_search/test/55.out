# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.97", "--temperature=1.5"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7000272274017334, "acc": 0.4995686113834381, "length": 3.61836838722229, "mode": "train", "epoch": 1}
{"loss": 0.695941686630249, "acc": 0.48460152745246887, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6969403624534607, "acc": 0.5219583511352539, "length": 3.54270076751709, "mode": "train", "epoch": 2}
{"loss": 0.6974749565124512, "acc": 0.4836956858634949, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6911020874977112, "acc": 0.5236514806747437, "length": 3.4033663272857666, "mode": "train", "epoch": 3}
{"loss": 0.69779372215271, "acc": 0.47826087474823, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6904147267341614, "acc": 0.5322155356407166, "length": 3.29451322555542, "mode": "train", "epoch": 4}
{"loss": 0.6982827186584473, "acc": 0.4809783101081848, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6884315013885498, "acc": 0.5420021414756775, "length": 3.0337953567504883, "mode": "train", "epoch": 5}
{"loss": 0.6964684128761292, "acc": 0.4900363087654114, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6874465346336365, "acc": 0.5368030071258545, "length": 2.7551348209381104, "mode": "train", "epoch": 6}
{"loss": 0.6949629783630371, "acc": 0.5090580582618713, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6792693734169006, "acc": 0.5628414154052734, "length": 2.5693368911743164, "mode": "train", "epoch": 7}
{"loss": 0.6847378015518188, "acc": 0.5291067361831665, "length": 1.29347825050354, "mode": "test", "epoch": 7}
{"loss": 0.6632307171821594, "acc": 0.6127325892448425, "length": 2.826442003250122, "mode": "train", "epoch": 8}
{"loss": 0.6507718563079834, "acc": 0.6598913669586182, "length": 2.4673912525177, "mode": "test", "epoch": 8}
{"loss": 0.6420559883117676, "acc": 0.6650388240814209, "length": 3.1112349033355713, "mode": "train", "epoch": 9}
{"loss": 0.6275447607040405, "acc": 0.7109172940254211, "length": 2.82608699798584, "mode": "test", "epoch": 9}
{"loss": 0.6038681268692017, "acc": 0.7147409319877625, "length": 3.1382179260253906, "mode": "train", "epoch": 10}
{"loss": 0.6050794124603271, "acc": 0.7272337079048157, "length": 2.9728260040283203, "mode": "test", "epoch": 10}
{"loss": 0.5650649666786194, "acc": 0.7258366942405701, "length": 3.265367031097412, "mode": "train", "epoch": 11}
{"loss": 0.585659384727478, "acc": 0.7470348477363586, "length": 3.1576087474823, "mode": "test", "epoch": 11}
{"loss": 0.5306527018547058, "acc": 0.7494295239448547, "length": 3.370828151702881, "mode": "train", "epoch": 12}
{"loss": 0.5824166536331177, "acc": 0.7371207475662231, "length": 3.3206522464752197, "mode": "test", "epoch": 12}
{"loss": 0.5116612315177917, "acc": 0.7668008804321289, "length": 3.4876890182495117, "mode": "train", "epoch": 13}
{"loss": 0.5701069831848145, "acc": 0.743011474609375, "length": 3.4728260040283203, "mode": "test", "epoch": 13}
{"loss": 0.4890797734260559, "acc": 0.7788166999816895, "length": 3.5037825107574463, "mode": "train", "epoch": 14}
{"loss": 0.5565536022186279, "acc": 0.7351251840591431, "length": 3.4076087474823, "mode": "test", "epoch": 14}
{"loss": 0.46309182047843933, "acc": 0.7842569351196289, "length": 3.5183587074279785, "mode": "train", "epoch": 15}
{"loss": 0.5758423209190369, "acc": 0.759346067905426, "length": 3.4891304969787598, "mode": "test", "epoch": 15}
{"loss": 0.4587611258029938, "acc": 0.7970649600028992, "length": 3.5540618896484375, "mode": "train", "epoch": 16}
{"loss": 0.5665029287338257, "acc": 0.7519491314888, "length": 3.4728260040283203, "mode": "test", "epoch": 16}
{"loss": 0.45066577196121216, "acc": 0.7987578511238098, "length": 3.5177578926086426, "mode": "train", "epoch": 17}
{"loss": 0.5599923729896545, "acc": 0.7705138921737671, "length": 3.36956524848938, "mode": "test", "epoch": 17}
{"loss": 0.44913822412490845, "acc": 0.8089826107025146, "length": 3.4769484996795654, "mode": "train", "epoch": 18}
{"loss": 0.5701992511749268, "acc": 0.7602903842926025, "length": 3.4728260040283203, "mode": "test", "epoch": 18}
{"loss": 0.4266204833984375, "acc": 0.8138595819473267, "length": 3.5608572959899902, "mode": "train", "epoch": 19}
{"loss": 0.5475685596466064, "acc": 0.7688649892807007, "length": 3.4347825050354004, "mode": "test", "epoch": 19}
{"loss": 0.41927239298820496, "acc": 0.8286060690879822, "length": 3.5442936420440674, "mode": "train", "epoch": 20}
{"loss": 0.5346032381057739, "acc": 0.7924864292144775, "length": 3.3478260040283203, "mode": "test", "epoch": 20}
{"loss": 0.39341768622398376, "acc": 0.841178834438324, "length": 3.5049588680267334, "mode": "train", "epoch": 21}
{"loss": 0.5214434266090393, "acc": 0.7928261756896973, "length": 3.4347825050354004, "mode": "test", "epoch": 21}
{"loss": 0.38175657391548157, "acc": 0.8419697880744934, "length": 3.5786452293395996, "mode": "train", "epoch": 22}
{"loss": 0.5160965919494629, "acc": 0.7846599221229553, "length": 3.4836957454681396, "mode": "test", "epoch": 22}
{"loss": 0.36913108825683594, "acc": 0.8440307378768921, "length": 3.6211256980895996, "mode": "train", "epoch": 23}
{"loss": 0.4927477240562439, "acc": 0.8024917840957642, "length": 3.53804349899292, "mode": "test", "epoch": 23}
{"loss": 0.36649563908576965, "acc": 0.8543742895126343, "length": 3.6572556495666504, "mode": "train", "epoch": 24}
{"loss": 0.4734203815460205, "acc": 0.8070420026779175, "length": 3.5706522464752197, "mode": "test", "epoch": 24}
{"loss": 0.3460981547832489, "acc": 0.8608596920967102, "length": 3.702296495437622, "mode": "train", "epoch": 25}
{"loss": 0.47559720277786255, "acc": 0.8143237829208374, "length": 3.6141304969787598, "mode": "test", "epoch": 25}
{"loss": 0.33815285563468933, "acc": 0.8596605062484741, "length": 3.726139783859253, "mode": "train", "epoch": 26}
{"loss": 0.47649797797203064, "acc": 0.8145027160644531, "length": 3.67391300201416, "mode": "test", "epoch": 26}
{"loss": 0.3316621780395508, "acc": 0.8707298636436462, "length": 3.7324178218841553, "mode": "train", "epoch": 27}
{"loss": 0.4678191840648651, "acc": 0.8312983512878418, "length": 3.5706522464752197, "mode": "test", "epoch": 27}
{"loss": 0.3208836019039154, "acc": 0.8710482716560364, "length": 3.6884565353393555, "mode": "train", "epoch": 28}
{"loss": 0.48970115184783936, "acc": 0.8142670392990112, "length": 3.66847825050354, "mode": "test", "epoch": 28}
{"loss": 0.3304264545440674, "acc": 0.8607572913169861, "length": 3.766646385192871, "mode": "train", "epoch": 29}
{"loss": 0.4701189696788788, "acc": 0.8276801705360413, "length": 3.6413042545318604, "mode": "test", "epoch": 29}
{"loss": 0.32231783866882324, "acc": 0.8707495927810669, "length": 3.7415263652801514, "mode": "train", "epoch": 30}
{"loss": 0.45239853858947754, "acc": 0.8337467908859253, "length": 3.6521739959716797, "mode": "test", "epoch": 30}
{"loss": 0.3074084222316742, "acc": 0.8752197027206421, "length": 3.7693212032318115, "mode": "train", "epoch": 31}
{"loss": 0.44998377561569214, "acc": 0.8282806277275085, "length": 3.71195650100708, "mode": "test", "epoch": 31}
{"loss": 0.31324467062950134, "acc": 0.8789443373680115, "length": 3.797966718673706, "mode": "train", "epoch": 32}
{"loss": 0.46687233448028564, "acc": 0.8223041296005249, "length": 3.75543475151062, "mode": "test", "epoch": 32}
{"loss": 0.3133382797241211, "acc": 0.8812173008918762, "length": 3.818023443222046, "mode": "train", "epoch": 33}
{"loss": 0.4573289155960083, "acc": 0.8331069350242615, "length": 3.6902174949645996, "mode": "test", "epoch": 33}
{"loss": 0.2959553003311157, "acc": 0.8833323121070862, "length": 3.78098201751709, "mode": "train", "epoch": 34}
{"loss": 0.4510553479194641, "acc": 0.8297414183616638, "length": 3.6847825050354004, "mode": "test", "epoch": 34}
{"loss": 0.2968023717403412, "acc": 0.8821232914924622, "length": 3.8037960529327393, "mode": "train", "epoch": 35}
{"loss": 0.44870877265930176, "acc": 0.8250547647476196, "length": 3.6847825050354004, "mode": "test", "epoch": 35}
{"loss": 0.28658249974250793, "acc": 0.8926388025283813, "length": 3.818331480026245, "mode": "train", "epoch": 36}
{"loss": 0.44792675971984863, "acc": 0.8366522789001465, "length": 3.6793477535247803, "mode": "test", "epoch": 36}
{"loss": 0.3020040690898895, "acc": 0.8941217660903931, "length": 3.8200855255126953, "mode": "train", "epoch": 37}
{"loss": 0.44303420186042786, "acc": 0.8338833451271057, "length": 3.70108699798584, "mode": "test", "epoch": 37}
{"loss": 0.2756848633289337, "acc": 0.8932668566703796, "length": 3.8188059329986572, "mode": "train", "epoch": 38}
{"loss": 0.4389091730117798, "acc": 0.8327327966690063, "length": 3.6956522464752197, "mode": "test", "epoch": 38}
{"loss": 0.2794003486633301, "acc": 0.8953219652175903, "length": 3.81449294090271, "mode": "train", "epoch": 39}
{"loss": 0.44788530468940735, "acc": 0.8368695974349976, "length": 3.70108699798584, "mode": "test", "epoch": 39}
{"loss": 0.2711223065853119, "acc": 0.8936384320259094, "length": 3.793635606765747, "mode": "train", "epoch": 40}
{"loss": 0.44011181592941284, "acc": 0.8433071374893188, "length": 3.6467392444610596, "mode": "test", "epoch": 40}
{"loss": 0.2812328338623047, "acc": 0.895929217338562, "length": 3.756882905960083, "mode": "train", "epoch": 41}
{"loss": 0.41020667552948, "acc": 0.8417342901229858, "length": 3.6576087474823, "mode": "test", "epoch": 41}
{"loss": 0.26306456327438354, "acc": 0.8972879648208618, "length": 3.788451910018921, "mode": "train", "epoch": 42}
{"loss": 0.4067329168319702, "acc": 0.8470496535301208, "length": 3.67391300201416, "mode": "test", "epoch": 42}
{"loss": 0.25614356994628906, "acc": 0.8972412347793579, "length": 3.813429594039917, "mode": "train", "epoch": 43}
{"loss": 0.43135207891464233, "acc": 0.838214099407196, "length": 3.7282607555389404, "mode": "test", "epoch": 43}
{"loss": 0.2533988654613495, "acc": 0.8969401121139526, "length": 3.8313117027282715, "mode": "train", "epoch": 44}
{"loss": 0.40227025747299194, "acc": 0.8489997982978821, "length": 3.70108699798584, "mode": "test", "epoch": 44}
{"loss": 0.2503584921360016, "acc": 0.9007819890975952, "length": 3.8207733631134033, "mode": "train", "epoch": 45}
{"loss": 0.4024888575077057, "acc": 0.8527321219444275, "length": 3.70652174949646, "mode": "test", "epoch": 45}
{"loss": 0.24826984107494354, "acc": 0.9066644906997681, "length": 3.812068462371826, "mode": "train", "epoch": 46}
{"loss": 0.39536386728286743, "acc": 0.8590848445892334, "length": 3.70652174949646, "mode": "test", "epoch": 46}
{"loss": 0.2349269837141037, "acc": 0.9083660840988159, "length": 3.8103532791137695, "mode": "train", "epoch": 47}
{"loss": 0.39777857065200806, "acc": 0.8547550439834595, "length": 3.6956522464752197, "mode": "test", "epoch": 47}
{"loss": 0.24750860035419464, "acc": 0.9098458290100098, "length": 3.801772356033325, "mode": "train", "epoch": 48}
{"loss": 0.4121624827384949, "acc": 0.8580244183540344, "length": 3.66304349899292, "mode": "test", "epoch": 48}
{"loss": 0.23541270196437836, "acc": 0.9036057591438293, "length": 3.7923882007598877, "mode": "train", "epoch": 49}
{"loss": 0.41354870796203613, "acc": 0.8480038642883301, "length": 3.6576087474823, "mode": "test", "epoch": 49}
{"loss": 0.24417726695537567, "acc": 0.9088822603225708, "length": 3.7893550395965576, "mode": "train", "epoch": 50}
{"loss": 0.42361557483673096, "acc": 0.8563008308410645, "length": 3.66304349899292, "mode": "test", "epoch": 50}
{"loss": 0.2397110015153885, "acc": 0.906535804271698, "length": 3.7826921939849854, "mode": "train", "epoch": 51}
{"loss": 0.390051007270813, "acc": 0.8592779636383057, "length": 3.66847825050354, "mode": "test", "epoch": 51}
{"loss": 0.2470664083957672, "acc": 0.9046289920806885, "length": 3.7908473014831543, "mode": "train", "epoch": 52}
{"loss": 0.3942451477050781, "acc": 0.8638119697570801, "length": 3.6793477535247803, "mode": "test", "epoch": 52}
{"loss": 0.2503809630870819, "acc": 0.9073282480239868, "length": 3.788088083267212, "mode": "train", "epoch": 53}
{"loss": 0.4341587424278259, "acc": 0.8472639918327332, "length": 3.66847825050354, "mode": "test", "epoch": 53}
{"loss": 0.22051945328712463, "acc": 0.9119181036949158, "length": 3.772644281387329, "mode": "train", "epoch": 54}
{"loss": 0.4004541039466858, "acc": 0.8524783849716187, "length": 3.625, "mode": "test", "epoch": 54}
{"loss": 0.23399405181407928, "acc": 0.9145535826683044, "length": 3.757556438446045, "mode": "train", "epoch": 55}
{"loss": 0.37143468856811523, "acc": 0.859035313129425, "length": 3.6576087474823, "mode": "test", "epoch": 55}
{"loss": 0.2218749076128006, "acc": 0.9100707173347473, "length": 3.790381908416748, "mode": "train", "epoch": 56}
{"loss": 0.3878612220287323, "acc": 0.8586795926094055, "length": 3.61956524848938, "mode": "test", "epoch": 56}
{"loss": 0.24004752933979034, "acc": 0.9154365062713623, "length": 3.766892671585083, "mode": "train", "epoch": 57}
{"loss": 0.3862406611442566, "acc": 0.8634430170059204, "length": 3.61956524848938, "mode": "test", "epoch": 57}
{"loss": 0.22347915172576904, "acc": 0.9131981134414673, "length": 3.7748708724975586, "mode": "train", "epoch": 58}
{"loss": 0.3867257833480835, "acc": 0.8602281808853149, "length": 3.625, "mode": "test", "epoch": 58}
{"loss": 0.22003793716430664, "acc": 0.9150385856628418, "length": 3.771789312362671, "mode": "train", "epoch": 59}
{"loss": 0.40601497888565063, "acc": 0.8526048064231873, "length": 3.6358695030212402, "mode": "test", "epoch": 59}
{"loss": 0.21463902294635773, "acc": 0.9197454452514648, "length": 3.804469108581543, "mode": "train", "epoch": 60}
{"loss": 0.3844790458679199, "acc": 0.8595427870750427, "length": 3.6847825050354004, "mode": "test", "epoch": 60}
{"loss": 0.22479143738746643, "acc": 0.9184965491294861, "length": 3.807840585708618, "mode": "train", "epoch": 61}
{"loss": 0.38528668880462646, "acc": 0.8661584258079529, "length": 3.67391300201416, "mode": "test", "epoch": 61}
{"loss": 0.21413922309875488, "acc": 0.9217302203178406, "length": 3.8137099742889404, "mode": "train", "epoch": 62}
{"loss": 0.386424720287323, "acc": 0.8588553667068481, "length": 3.66847825050354, "mode": "test", "epoch": 62}
{"loss": 0.20410816371440887, "acc": 0.9177036881446838, "length": 3.811490535736084, "mode": "train", "epoch": 63}
{"loss": 0.37218648195266724, "acc": 0.8725272417068481, "length": 3.63043475151062, "mode": "test", "epoch": 63}
{"loss": 0.2181241810321808, "acc": 0.9223453402519226, "length": 3.7863526344299316, "mode": "train", "epoch": 64}
{"loss": 0.39211615920066833, "acc": 0.8683874607086182, "length": 3.6467392444610596, "mode": "test", "epoch": 64}
{"loss": 0.21706543862819672, "acc": 0.9191770553588867, "length": 3.793781042098999, "mode": "train", "epoch": 65}
{"loss": 0.37549448013305664, "acc": 0.8620427846908569, "length": 3.6576087474823, "mode": "test", "epoch": 65}
{"loss": 0.20007780194282532, "acc": 0.9249812364578247, "length": 3.8192543983459473, "mode": "train", "epoch": 66}
{"loss": 0.381772518157959, "acc": 0.8711311221122742, "length": 3.67391300201416, "mode": "test", "epoch": 66}
{"loss": 0.20592761039733887, "acc": 0.9249345064163208, "length": 3.824493408203125, "mode": "train", "epoch": 67}
{"loss": 0.3822495937347412, "acc": 0.8709259033203125, "length": 3.6902174949645996, "mode": "test", "epoch": 67}
{"loss": 0.2092808336019516, "acc": 0.9257805943489075, "length": 3.809649705886841, "mode": "train", "epoch": 68}
{"loss": 0.3765701651573181, "acc": 0.8650968670845032, "length": 3.6358695030212402, "mode": "test", "epoch": 68}
{"loss": 0.20833753049373627, "acc": 0.9241036176681519, "length": 3.7620253562927246, "mode": "train", "epoch": 69}
{"loss": 0.3663872182369232, "acc": 0.8725019693374634, "length": 3.5978260040283203, "mode": "test", "epoch": 69}
{"loss": 0.1936139315366745, "acc": 0.9285828471183777, "length": 3.7900912761688232, "mode": "train", "epoch": 70}
{"loss": 0.3666139245033264, "acc": 0.8760402798652649, "length": 3.6358695030212402, "mode": "test", "epoch": 70}
{"loss": 0.18643765151500702, "acc": 0.9237497448921204, "length": 3.7945477962493896, "mode": "train", "epoch": 71}
{"loss": 0.40448129177093506, "acc": 0.8696338534355164, "length": 3.6141304969787598, "mode": "test", "epoch": 71}
{"loss": 0.2136300653219223, "acc": 0.9172657132148743, "length": 3.775651693344116, "mode": "train", "epoch": 72}
{"loss": 0.36534470319747925, "acc": 0.8783653974533081, "length": 3.625, "mode": "test", "epoch": 72}
{"loss": 0.19881510734558105, "acc": 0.9249271750450134, "length": 3.770217180252075, "mode": "train", "epoch": 73}
{"loss": 0.3778257966041565, "acc": 0.88136887550354, "length": 3.6032607555389404, "mode": "test", "epoch": 73}
{"loss": 0.20310401916503906, "acc": 0.9246841669082642, "length": 3.7521941661834717, "mode": "train", "epoch": 74}
{"loss": 0.3814404010772705, "acc": 0.8783805966377258, "length": 3.6032607555389404, "mode": "test", "epoch": 74}
{"loss": 0.1897747814655304, "acc": 0.9265508651733398, "length": 3.7791192531585693, "mode": "train", "epoch": 75}
{"loss": 0.3747355341911316, "acc": 0.878360390663147, "length": 3.63043475151062, "mode": "test", "epoch": 75}
{"loss": 0.19845327734947205, "acc": 0.9260084629058838, "length": 3.792271375656128, "mode": "train", "epoch": 76}
{"loss": 0.42594459652900696, "acc": 0.8634855151176453, "length": 3.625, "mode": "test", "epoch": 76}
{"loss": 0.2097788155078888, "acc": 0.9169754385948181, "length": 3.7623205184936523, "mode": "train", "epoch": 77}
{"loss": 0.42793241143226624, "acc": 0.868270218372345, "length": 3.6032607555389404, "mode": "test", "epoch": 77}
{"loss": 0.18966570496559143, "acc": 0.9271995425224304, "length": 3.777169704437256, "mode": "train", "epoch": 78}
{"loss": 0.39609503746032715, "acc": 0.8672400712966919, "length": 3.5978260040283203, "mode": "test", "epoch": 78}
{"loss": 0.19125603139400482, "acc": 0.925493597984314, "length": 3.7712574005126953, "mode": "train", "epoch": 79}
{"loss": 0.39155396819114685, "acc": 0.8737079501152039, "length": 3.5978260040283203, "mode": "test", "epoch": 79}
{"loss": 0.18001008033752441, "acc": 0.9273494482040405, "length": 3.7525012493133545, "mode": "train", "epoch": 80}
{"loss": 0.3955965042114258, "acc": 0.8744379878044128, "length": 3.625, "mode": "test", "epoch": 80}
{"loss": 0.16893456876277924, "acc": 0.9319093823432922, "length": 3.792055130004883, "mode": "train", "epoch": 81}
{"loss": 0.3698781728744507, "acc": 0.8787708878517151, "length": 3.67391300201416, "mode": "test", "epoch": 81}
{"loss": 0.18024705350399017, "acc": 0.9355162978172302, "length": 3.822638750076294, "mode": "train", "epoch": 82}
{"loss": 0.3650965094566345, "acc": 0.8807845711708069, "length": 3.6086957454681396, "mode": "test", "epoch": 82}
{"loss": 0.18202924728393555, "acc": 0.9351971745491028, "length": 3.780700922012329, "mode": "train", "epoch": 83}
{"loss": 0.35833728313446045, "acc": 0.8785887956619263, "length": 3.625, "mode": "test", "epoch": 83}
{"loss": 0.1899457722902298, "acc": 0.9357333183288574, "length": 3.8124566078186035, "mode": "train", "epoch": 84}
{"loss": 0.3794413208961487, "acc": 0.8788011074066162, "length": 3.6847825050354004, "mode": "test", "epoch": 84}
{"loss": 0.18061621487140656, "acc": 0.9317899346351624, "length": 3.7971794605255127, "mode": "train", "epoch": 85}
{"loss": 0.3622645437717438, "acc": 0.8756519556045532, "length": 3.6141304969787598, "mode": "test", "epoch": 85}
{"loss": 0.17724071443080902, "acc": 0.932908296585083, "length": 3.797105073928833, "mode": "train", "epoch": 86}
{"loss": 0.348042756319046, "acc": 0.882906436920166, "length": 3.66304349899292, "mode": "test", "epoch": 86}
{"loss": 0.18416032195091248, "acc": 0.9320188164710999, "length": 3.8198671340942383, "mode": "train", "epoch": 87}
{"loss": 0.3665517568588257, "acc": 0.8751739263534546, "length": 3.67391300201416, "mode": "test", "epoch": 87}
{"loss": 0.1752193719148636, "acc": 0.9365956783294678, "length": 3.8025949001312256, "mode": "train", "epoch": 88}
{"loss": 0.39019280672073364, "acc": 0.8750646114349365, "length": 3.625, "mode": "test", "epoch": 88}
{"loss": 0.1839199811220169, "acc": 0.9304903745651245, "length": 3.7763609886169434, "mode": "train", "epoch": 89}
{"loss": 0.3820456266403198, "acc": 0.8834586143493652, "length": 3.6793477535247803, "mode": "test", "epoch": 89}
{"loss": 0.1663123220205307, "acc": 0.9370566010475159, "length": 3.823946714401245, "mode": "train", "epoch": 90}
{"loss": 0.37707388401031494, "acc": 0.8868217468261719, "length": 3.6902174949645996, "mode": "test", "epoch": 90}
{"loss": 0.18354018032550812, "acc": 0.935259997844696, "length": 3.802471160888672, "mode": "train", "epoch": 91}
{"loss": 0.38235294818878174, "acc": 0.876799464225769, "length": 3.6032607555389404, "mode": "test", "epoch": 91}
{"loss": 0.17224107682704926, "acc": 0.9388006925582886, "length": 3.7801315784454346, "mode": "train", "epoch": 92}
{"loss": 0.35473310947418213, "acc": 0.8840872645378113, "length": 3.6141304969787598, "mode": "test", "epoch": 92}
{"loss": 0.17282317578792572, "acc": 0.9393312931060791, "length": 3.7920844554901123, "mode": "train", "epoch": 93}
{"loss": 0.3932393789291382, "acc": 0.8763931393623352, "length": 3.6576087474823, "mode": "test", "epoch": 93}
{"loss": 0.1857694834470749, "acc": 0.9309621453285217, "length": 3.7944459915161133, "mode": "train", "epoch": 94}
{"loss": 0.38275378942489624, "acc": 0.8760412335395813, "length": 3.66304349899292, "mode": "test", "epoch": 94}
{"loss": 0.16728650033473969, "acc": 0.9379733204841614, "length": 3.7989518642425537, "mode": "train", "epoch": 95}
{"loss": 0.3743361830711365, "acc": 0.8814154863357544, "length": 3.67391300201416, "mode": "test", "epoch": 95}
{"loss": 0.17815427482128143, "acc": 0.937969982624054, "length": 3.824225902557373, "mode": "train", "epoch": 96}
{"loss": 0.35176634788513184, "acc": 0.8824242949485779, "length": 3.70108699798584, "mode": "test", "epoch": 96}
{"loss": 0.1752004474401474, "acc": 0.9338563084602356, "length": 3.8131003379821777, "mode": "train", "epoch": 97}
{"loss": 0.35583949089050293, "acc": 0.879977822303772, "length": 3.70652174949646, "mode": "test", "epoch": 97}
{"loss": 0.1576804667711258, "acc": 0.940462052822113, "length": 3.8083677291870117, "mode": "train", "epoch": 98}
{"loss": 0.4019642472267151, "acc": 0.8756793737411499, "length": 3.6956522464752197, "mode": "test", "epoch": 98}
{"loss": 0.15505342185497284, "acc": 0.9441651105880737, "length": 3.814098596572876, "mode": "train", "epoch": 99}
{"loss": 0.3906440734863281, "acc": 0.8811759352684021, "length": 3.70108699798584, "mode": "test", "epoch": 99}
{"loss": 0.15393590927124023, "acc": 0.9457260966300964, "length": 3.8028743267059326, "mode": "train", "epoch": 100}
{"loss": 0.35027483105659485, "acc": 0.8852134943008423, "length": 3.6847825050354004, "mode": "test", "epoch": 100}
test accuracy: 0.740732729434967
